src/editor/editor_layer.cpp

```
@@ -73,6 +74,17 @@ void EditorLayer::Update(uint32_t ticks) {
         panel->Update(ticks);
     }
 
+    if (m_config.AutoSaveEnabled && !m_currentLayoutPath.empty() && IsWorkPending()) {
+        m_autoSaveAccumulatedMs += ticks;
+        uint32_t const intervalMs = static_cast<uint32_t>(m_config.AutoSaveIntervalMinutes) * 60u * 1000u;
```

Clang (14.0.6)

[error] variable 'intervalMs' is not initialized

(cppcoreguidelines-init-variables,-warnings-as-errors)